### PR TITLE
[PRM-1591] - Redesigned timeline view for compliance page

### PR DESCRIPTION
**Pull request by LewisMitchell**
image

### CODE REVIEW

File: app/models/lsp/LSPSummary.scala [hunk 1]
```diff
@@ -18,11 +18,14 @@ package models.lsp
...
+import java.time.LocalDate
+
 case class LSPSummary(
                        activePenaltyPoints: Int,
                        inactivePenaltyPoints: Int,
                        regimeThreshold: Int,
-                       penaltyChargeAmount: BigDecimal
+                       penaltyChargeAmount: BigDecimal,
+                       PoCAchievementDate: LocalDate
                      )
```
Comment (ruthdas03) [inline app/models/lsp/LSPSummary.scala:29]: Do we not want to add 'PoCAchievementDate' to LSPSummary model in backend

Comment (LewisMitchell) [reply app/models/lsp/LSPSummary.scala:29]: Already done here: https://github.com/hmrc/penalties/pull/169

Comment (ruthdas03) [reply app/models/lsp/LSPSummary.scala:29]: > Already done here: [hmrc/penalties#169](https://github.com/hmrc/penalties/pull/169)

Ah! sorry I missed it.

Comment (ruthdas03) [reply app/models/lsp/LSPSummary.scala:29]: Runs ok against the AT branch PRM-1594

**Review by ruthdas03 — APPROVED** [review]

**Pull request merged**